src/sentry/api/serializers/models/release.py

```diff
@@ -256,6 +257,7 @@ def serialize(self, obj, attrs, user, *args, **kwargs):
         d = {
             'version': obj.version,
             'shortVersion': obj.short_version,
+            'safeVersion': quote(obj.version, safe=''),
```

Comment:
Open to other names for this if anyone has opinions

Reply:
Some other options could be:

* slug
* urlVersion
* pathName

Reply:
Definitely not `safeVersion`

Slug is what we use elsewhere (e.g. `project.slug`).

Reply:
`slug` sounds like the best option